### DISCUSSION TITLE
Notifications URL

### DISCUSSION THREAD

**Discussion by excaliburzarau** [offen/docker-volume-backup#237] [Q&A] [answered]
Hello guys, 

I have been trying to setup the Notification URL but i am not able to see how can i configure an hotmail email to send the notifications. I have it already setup on a different application and it is working fine. I believe it might be possible that the authentication method is incorrect but i already tried them all.

This is the Env Variable i am using:

- NOTIFICATION_URLS=smtp://senderemail%40hotmail.com:password123@smtp.office365.com:587/?fromAddress=senderemail@hotmail.com&toAddresses=destinyemail@hotmail.com&auth=Unknown&starttls=Yes

I do believe this should be enough by reading the Doc but i am unable to make it successfully send the email.
Can you guys give me some pointers?
The error i receive is the one bellow.

Backup procedure ran successfully, but an error ocurred calling the registered hooks: runHooks: error running hook: notify: error notifying: sendNotification: error sending message: failed to send using smtp: timed out"

Thank you all in advance.

**Comment by m90** [comment]
It looks to me as if your query parameters are not URL encoded (i.e. they contain a literal `@`. Did you use the `shoutrrr` tool to generate the URL as described here: https://github.com/offen/docker-volume-backup#use-special-characters-in-notification-urls ?

**Reply by excaliburzarau** [reply]
Thank you for your reply.

I was not using the tool to generate and indeed the @ does resolve for a %40 except after the smtp host. 
I was not using the %40 on all of them but it the issue does persist. 
I have tried to figure out the configuration for the Outlook IMAP regarding the Authentication, ClientHost and Encryption but without success. I set the Authentication to Unknown, the ClientHost to Auto and the Encryption also to Auto but it does not seem to be working for some reason.

Thank you in advance for all your help.
Best Regards

**Reply by m90** [reply, marked as answer]
Maybe it would be worth it digging through the issues in the shoutrrr repository as they might have much more usage than here and you might also encounter someone using it with Hotmail?

**Reply by excaliburzarau** [reply]
I will try to do so.

Thank you for all your help.
